### PR TITLE
Homework22: Pavel Mitrafanau

### PR DESCRIPTION
**Pull request by Pavel2812mi**
Good day, Sergei
My robot framework tests are ready for review

### CODE REVIEW

File: library_keywords.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+def create_book(name, author, number_of_pages, ISBN, reserved):
+    """Create a new book"""
+    return Book(name=name, author=author, number_of_pages=number_of_pages,
+                ISBN=ISBN, reserved=reserved)
+
+
+@keyword
+def create_reader(name):
+    """Create a new reader"""
+    return Reader(name=name)
```
Comment (Stml89) [inline library_keywords.py]:
```suggestion
    return Reader(name)
```

**Review by Stml89 — APPROVED** [review]

File: bank_deposit_robot.robot
Comment (Stml89) [inline bank_deposit_robot.robot:1]: Can not find any logging here

File: library_keywords.py
Comment (Stml89) [inline library_keywords.py:1]: Can not find any logging here

**Review by Stml89 — CHANGES_REQUESTED** [review]

**Comment by Pavel2812mi** [comment]
Logging added to library and bank deposit tests

**Review by Stml89 — APPROVED** [review]
Good project structure: https://docs.robotframework.org/docs/examples/project_structure#examples

**Pull request merged**